utils/installer/install.sh

```diff
@@ -439,6 +439,7 @@ function setup_lvim() {
 
   "$INSTALL_PREFIX/bin/$NVIM_APPNAME" --headless -c 'quitall'
 
+  echo ""
   echo "Lazy setup complete"
```

Comment:
Not sure, but I think we could use `printf` here.

```suggestion
  printf "\nLazy setup complete"
```